src/viewer.ts

```diff
@@ -500,12 +506,11 @@ class Viewer {
             //     name: 'heli_equirect',
             //     width: 2048,
             //     height: 1024,
-            //     type: pc.TEXTURETYPE_RGBM
+            //     type: pc.TEXTURETYPE_RGBM.toString()
             // });
-            // pc.reprojectTexture(app.graphicsDevice, cubemap.resource, equi);
-            // pc.downloadTexture(equi, 'Helipad_equi.png', 0, true);
-
-            // pc.downloadTexture(cubemap.resource, 'Helipad_cube.png');
+            // pc.reprojectTexture(app.graphicsDevice, cubemap.resources[1], equi);
+            // downloadTexture(equi, 'Helipad_equi.png', 0, true);
+            // downloadTexture(cubemap.resources[1], 'Helipad_cube.png');
```

Comment:
Should all this commented out code just be removed?

Reply:
Gotcha

Reply:
I left it there because I find it useful for debugging. If you're super keen to see it go though, I can make it so.

Reply:
Nah, it's fine. I was just asking because the default should be to avoid checking in commented out code. I'll resolve this...